styles/addMember.css

```diff
@@ -0,0 +1,52 @@
+form {
+    margin-left: 145px;
+    margin-top: 48px;
+  }
+  
+input[type=text], input[type=date] {
+    width: 576px;
+    height:43px;
+    padding: 12px 16px;
+    align-items: center;
+    margin: 8px 0;
+    border: 2px solid var(--clr-white);
+    border-radius: 7px;
+    gap: 10px;
+    box-sizing: border-box;
+  }
+
+ button  {
+     color: var(--clr-white);
+    padding: 12px 24px;
+    border: none;
+    border-radius: 7px;
+    font-weight: 600;
+    margin-left: 428px;
+    margin-top:32px;
+
```

Comment:
You had the right idea of using `margin` to push the button to the right, but the idea is to have the button _align itself_ to the right of the form. What if the form got stretched? The button would still need to move accordingly right?

So I'll introduce you to flexbox. You can have a read [here](https://css-tricks.com/snippets/css/a-guide-to-flexbox/). But for now here's what you need to do.

- give your form a class, and then select that class and give it this style (I'll annotate them for explanation)
```css
.your-form-class-name {
  display: flex; /* Enable flexbox styling */
  flex-direction: column; /* Layout all of its children from top to bottom. The default value is 'row', ie. left to right */
  width: fit-content; /* Make sure the form's width matches the longest child's width and not longer. I will explain more in the 2nd step */
}
```
- then, give your button an id, then give it an `align-self: flex-end;` style
```css
#your-form-button {
  align-self: flex-end;  /* Flex-end in a flex-direction: column is to the right, so you're saying "the button should align itself to the right of the parent". The reason we set the width to fit the longest child (in this case the input) is because that's where the button should always be, to the right of the input */
}
```